src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The link to your stylesheet should point to a CSS file, not a SCSS file. The browser cannot interpret SCSS syntax. Make sure you compile your SCSS to CSS and then link that file here.